OxfordBodleian/Ullendorff/BDLeu4.xml

```diff
@@ -173,37 +173,78 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     </dimensions>
                                 </layout>
                             </layoutDesc>
-                            
+
                         </objectDesc>
-                        
+
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
-                           
+
                             </handNote>
                         </handDesc>
-                        
+
                        <!-- <decoDesc>
-                         
+
                         </decoDesc>-->
-                        
-                   <!--     <bindingDesc>
-                            <binding xml:id="binding">
-                                <decoNote xml:id="b1"></decoNote>
-                                <decoNote xml:id="b2" type="bindingMaterial">
-                           <material key="wood"/>
-                        </decoNote>
+
+                        <bindingDesc>
+                            <binding xml:id="binding" contemporary="true Ethiopian">
+                              <p>
+                                The manuscript was repaired several times, as testify the various threads visible on the spine and holes drilled through the boards.
```

Comment:
I would say "as testified by the various threads visible on the spine and holes drilled through the boards." But you can change this when we do the editing